tests/it/e2e_rpc_server.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::Ok;
+
+use crate::util::{wasm_file, Soroban};
+
+// e2e tests are ignore by default
+#[test]
+#[ignore]
+fn e2e_deploy_and_invoke_contract_against_rpc_server() -> Result<(), anyhow::Error> {
```

Comment:
❔ I'm not opposed to using anyhow in applications, but what do we get out of adding that dependency here? It appears we're only replacing `dyn std::error::Error` with `anyhow::Error`, and replacing `Result::Ok` with `anyhow::Ok`.

Reply:
Ah I see, it was added for the wasm_file fn. I think that the file-doesn't-exist case can continue to panic. There's little advantage as far as I know to generating errors in tests as they're presented in the same format/output as panics are in the test output. In a step-through debugger a panic is also captured typically which makes debugging test setup/utility failures easier.

Reply:
Removed Results in favor of panics.